log-base/src/Log/Monad.hs

```diff
@@ -42,6 +45,24 @@ newtype LogT m a = LogT { unLogT :: InnerLogT m a }
   deriving (Alternative, Applicative, Functor, Monad, MonadBase b, MonadCatch
            ,MonadIO, MonadMask, MonadPlus, MonadThrow, MonadTrans)
 
+instance MonadError e m => MonadError e (LogT m) where
+    throwError = lift . throwError
+    catchError m h = LogT $ unLogT m `catchError` (unLogT . h)
+
+instance MonadReader r m => MonadReader r (LogT m) where
```

Comment:
I'm :-1:, it's a bit like writing of `MonadReader s (StateT m) where ask = get`, correct but weird. I'd rather leave ability to write `LogT (ReaderT r m) a` for people if they want to do so.